package.json

```
@@ -151,6 +151,22 @@
           "type": "string",
           "default": "",
           "description": "Manually set a language server executable. Can be something on the $PATH or a path to an executable itself. Works with ~, ${HOME} and ${workspaceFolder}."
+        },
+        "haskell.hlsUpdateBehavior": {
```

One last nitpick sorry, the title gets rendered weirdly in VS Code like "Hls Update Behaviour", I think we can just assume the default is HLS and rename this
```suggestion
        "haskell.updateBehavior": {
```